js/src/components/raise-budget-recommendation-banner/index.test.js

```diff
@@ -0,0 +1,292 @@
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom';
+import { getHistory } from '@woocommerce/navigation';
+import { useDispatch } from '@wordpress/data';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { getEditCampaignUrl } from '~/utils/urls';
+import { PREFERENCES_STORE_NAMESPACE, DAY_IN_SECONDS } from '~/constants';
+import RaiseBudgetRecommendationBanner from './index';
+import usePreference from '~/hooks/usePreference';
+import useRaiseBudgetRecommendations from '~/hooks/useRaiseBudgetRecommendations';
+import useGoogleAdsAccountReady from '~/hooks/useGoogleAdsAccountReady';
+import useAdsCampaigns from '~/hooks/useAdsCampaigns';
+import mockedRecommendedCampaigns from '../../../../tests/mocks/ads/recommendations/raise-buget-recommendations.json';
```

Comment:
note `buget` missing the `d`.

Reply:
Fixed in [e452cd8](https://github.com/woocommerce/google-listings-and-ads/pull/3133/commits/e452cd81ee74a92273686486298e0b50f1110642)